submissions/bilkisuTace/html-forms/contact.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Contact | Ladies & Gents</title>
+</head>
+<body>
+
+	<header>
+
+		<h1>LADIES & GENTS</h1>
+		
```

Comment:
There are too many extra spaces in this file. I think you should remove them for readability @bilkisutace